### ISSUE TITLE
Automate add deploys.json with Rinkeby and Mainnet deploys

### ISSUE THREAD

**Issue by poiresel** [airswap/airswap-protocols#255]
Create and add a deploys.json to the repository. 

deploys.json will contain a json object of all deploys to mainnet and rinkeby
Future work will be automating and using this to create a markdown (human friendly page)

**Comment by alicevhenshaw** [comment]
Isn't this essentially the same issue as https://github.com/airswap/airswap-protocols/issues/250

**Comment by dmosites** [comment]
Propose that we create a deploys.json for each package that includes the respective Rinkeby and Mainnet deploys for that package version.

**Comment by poiresel** [comment]
proposed deploys.json format
{"rinkeby": 
 {
   {"addresses": 
     {
       {
         "CONTRACT_NAME": "ADDRESS",
         "CONTRACT_NAME": "ADDRESS"
       }
    }
 }
}

**Comment by ejwessel** [comment]
`truffle migrate` has a `--describe-json` option

```
{
  "status": "deployed",
  "data": {
    "contract": {
      "contractName": "Types",
      "address": "0x1CE4504E0736Cc88c7c9177Be09F2b91763A79DA"
    },
    "deployed": true,
    "receipt": {
      "transactionHash": "0xad91739d8ec999b9c961bfc2e088c10bb62170222019d8fac31f457df08a9975",
      "gasUsed": 996609
    },
    "timestamp": 1572892659,
    "gasPrice": "2",
    "gas": "996609",
    "from": "0xb893D8F6779842959C1dfC3095b1c62ceAA16703",
    "value": "0",
    "cost": "0.001993218",
    "balance": "12.184292564"
  }
}```

**Comment by dmosites** [comment]
Propose the following format for `deploys.json`:

```
{
  <networkId>: <address>,
  ...
}
```

**Comment by ejwessel** [comment]
I don't think my PR will solve everything, but we could always move to https://www.npmjs.com/package/truffle-deploy-registry

**Comment by ejwessel** [comment]
https://github.com/airswap/airswap-protocols/pull/475

**Comment by ejwessel** [comment]
addressed in #483
